src/components/ModalNew.tsx

```diff
@@ -0,0 +1,166 @@
+import { colorsV2 } from '@hedviginsurance/brand'
+import { motion } from 'framer-motion'
+import hexToRgba from 'hex-to-rgba'
+import * as React from 'react'
+import styled from 'react-emotion'
+import { Cross } from './icons/Cross'
+
+export interface ModalProps {
+  isVisible: boolean
+  onClose: () => void
+}
+
+const Wrapper = styled(motion.div)`
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  z-index: 2000;
+`
+
+const Background = styled(motion.div)`
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  background-color: ${hexToRgba(colorsV2.white, 0.75)};
+`
+
+const Container = styled(motion.div)`
```

Comment:
We should probably name this `wrapper` just to not confuse it with  a state container :D also to be consistent with the rest of the code base

Reply:
Yeah, in the other context we don't use `constate` so that wasn't a problem there 😄